discord_slash/model.py

```diff
@@ -176,6 +175,28 @@ async def can_run(self, ctx) -> bool:
         return False not in res
 
 
+class BasecommandObject(CommandObject):
```

Comment:
Shouldn't this and it's dependents be named `BaseCommandObject`?

```suggestion
class BaseCommandObject(CommandObject):
```

Reply:
Previously it was`SubcommandObject` for sub commands, so naturally I named it `BasecommandObject`. But if the devs wants me to change it, then sure

Reply:
"Subcommand", is considered one word, that's what the API docs use. "BaseCommand" is two words.